packages/kit/test/apps/basics/test/test.js

```diff
@@ -2572,12 +2572,17 @@ test.describe('Static files', () => {
 		expect(response.status()).toBe(process.env.DEV ? 403 : 404);
 	});
 
-	test('Vite serves assets in src directory', async ({ page, request }) => {
+	test('Vite serves assets in allowed directories', async ({ page, request }) => {
 		await page.goto('/assets');
 		const path = await page.textContent('h1');
 
-		const response = await request.get(path);
-		expect(response.status()).toBe(200);
+		const r1 = await request.get(path);
+		expect(r1.status()).toBe(200);
+		expect(await r1.text()).toContain('http://www.w3.org/2000/svg');
+
+		const r2 = await request.get('/package.json');
```

Comment:
it'd be nice to add a comment for package.json since it's easy to miss what it's testing
```suggestion
		// check that we can fetch a route which overlaps with the name of a file
		const r2 = await request.get('/package.json');
```

that's not exactly what I was suggesting with `node_modules`, but you can ignore because I didn't correctly understand what this PR was doing